scripts/grpc-unit-test.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
How is this different from `scripts/grpc-integration-test.sh`? Our goal is to **simplify** and **consolidate** code, not duplicate it.